integrations/cli/config.test.ts

```diff
@@ -161,7 +161,10 @@ test(
     },
   },
   async ({ fs, spawn }) => {
-    await spawn('pnpm tailwindcss --input src/index.css --output dist/out.css --watch')
+    let process = await spawn(
+      'pnpm tailwindcss --input src/index.css --output dist/out.css --watch',
+    )
+    await process.onStderr((m) => m.includes('Done in'))
```

Comment:
All `spawn(…)` tests will wait for a specific message to ensure we wait for everything to be ready before making changes. This improves the reliability of the tests.